src/config.js

```diff
@@ -17,9 +17,9 @@ export function getConfig(mode) {
       'annot_color': '#3f51b5',
       'annot_url': getPhenoLinkTarget,
       'bio_link': BIOLINK + 'bioentityset/slimmer/phenotype?',
-      'evenRowColor': '#EAF0EF',
-      //'oddRowColor': '#FFFAE4',
-      'oddRowColor': '#ffffff',
+      // 'evenRowColor': '#EAF0EF',
```

Comment:
can these just be removed?

Reply:
Yes, I just let them in case while testing